ansible-tailscale-inventory.py

```diff
@@ -53,9 +53,10 @@
     if "Tags" in v:
         for tag in v["Tags"]:
             safe_tag = tag.replace(":", "_")
```

Comment:
Instead of adding the new var `final_tag`, what do you think about doing something like this to collapse the fix into a one-liner:
```suggestion
            safe_tag = tag.replace(":", "_").replace("-", "_")
```

Reply:
Well initally I wanted to do it that way but then my tests always failed. But I just tried again and seems like my tests were done with an ansible playbook that had some errors, the inventory itself looks just fine that way.

So yes that's of course the better way to do this.

Reply:
Awesome I'm glad it worked!